CMakeLists.txt

```diff
@@ -282,3 +282,24 @@ endif()
 set(docfiles COPYING NEWS AUTHORS)
 install(FILES ${docfiles}
         DESTINATION "${DOCDIR}")
+
+################################################################################
+# pkg-config support
+################################################################################
+if(UNIX)
```

Comment:
I just had nothing but MinGW (MSYS2) today, and guess what... There are other pkg-config files here. I successfully tested build/install with:
~~~
if(UNIX OR MINGW)
~~~

Reply:
hmm, possibly this one too, as some of these paths are set from `GNUInstallDirs` https://github.com/OSGeo/PROJ/blob/1a518773616fb001483c175e8dfab00c06e83c89/cmake/ProjInstallPath.cmake#L26-L27

Reply:
I see. I don't see a difference in `proj.pc` when I change `ProjInstallPath.cmake`.
And I wouldn't delay PROJ 8.0.0 for MINGW pkgconfig.

Default config for MinGW, generator Ninja:
~~~
prefix=c:/OSGeo4W
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
datarootdir=${prefix}/share
datadir=${prefix}/share/proj

Name: PROJ
Description: Coordinate transformation software library
Requires:
Version: 8.0.0
Libs: -L${libdir} -lproj
Libs.private: -lsqlite3 -ltiff -lcurl -lstdc++
Cflags: -I${includedir}
~~~

Looks good to me.